opentok/opentok.py

```diff
@@ -433,14 +448,120 @@ def get_archives(self, offset=None, count=None):
+
+    def signal(self, session_id, connection_id, payload):
+        """Sends a signal to all the connections in a session or to a specific one. This is the
+          server-side equivalent to the signal() method in the OpenTok client SDKs:
+          https://www.tokbox.com/developer/guides/signaling/js/.
+
+        :param String session_id: The session ID for the OpenTok session to send the signal to.
+        :param String connection_id: The connection ID of a client connected to the session. Leave
+          this empty if you want to send a signal to all connections in the session.
+        :param Dictionary payload: An object with optional signal type and signal data fields.
+        """
+        if not session_id or not payload:
+            raise OpenTokException(u('SessionId and payload are required'))
+
+        response = requests.post(self.connection_url(session_id, connection_id) + '/signal', data=json.dumps(payload), headers=self.api_headers(), proxies=self.proxies)
+
+        if response.status_code < 300:
+            pass
+        elif response.status_code == 403:
+            raise AuthError()
         elif response.status_code == 404:
-            raise NotFoundError("Archive not found")
+            raise NotFoundError("Session or connection not found")
+        else:
+            raise RequestError("An unexpected error occurred", response.status_code)
+
+    def force_disconnect(self, session_id, connection_id):
+        """Disconnects a participant from an OpenTok session. This is the server-side equivalent
+          to the forceDisconnect() method in OpenTok.js:
+          https://www.tokbox.com/developer/guides/moderation/js/#force_disconnect
+
+        :param String session_id: The session ID for the OpenTok session that the client you want
+          to disconnect is connected to.
+        :param String connection_id: The connection ID of the client you want to disconnect.
+        """
+        if not session_id or not connection_id:
+            raise OpenTokException(u('SessionId and ConnectionId are required'))
+
+        response = requests.delete(self.connection_url(session_id, connection_id), headers=self.api_headers(), proxies=self.proxies)
+
+        if response.status_code < 300:
+            pass
+        elif response.status_code == 403:
+            raise AuthError()
+        elif response.status_code == 404:
+            raise NotFoundError("Session or connection not found")
+        else:
+            raise RequestError("An unexpected error occurred", response.status_code)
+
+    def register_callback(self, group, event, url):
+        """Register a callback URL for a specific group and event to receive OpenTok Cloud API
+        events (webhooks) for your OpenTok API key.
+
+        :param String group: The group of events you are interested in. It can be set to 'archive',
+          'connection' or 'stream'.
+        :param String event: The specific event from the group you are interested on receiving
```

Comment:
Perhaps we should add an enum: CallbackEvents. Unless this could be an arbitrary string in the future.